.github/workflows/contributors.yml

```
@@ -10,13 +10,20 @@ jobs:
     permissions:
       contents: write
       pull-requests: write
+    env:
+      GH_TOKEN: ${{ secrets.GH_PAT }}
 
     steps:
```

actionlint (1.7.8)

could not parse as YAML: did not find expected '-' indicator

(syntax-check)